packages/web-app-files/src/components/SideBar/Shares/Links/ListItem.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="oc-width-1-1 oc-flex oc-flex-middle oc-flex-between files-links-details">
     <div class="oc-flex oc-flex-middle">
-      <oc-avatar-item :width="36" icon-size="medium" icon="link" name="df" />
+      <oc-avatar-item :width="36" icon-size="medium" icon="link" name="link" />
       <div class="files-links-name-wrapper oc-pl-s">
-        <div class="oc-flex oc-flex-middle">
-          <div class="oc-text-truncate">
-            <span aria-hidden="true" class="files-links-name" v-text="linkShare.displayName" />
-          </div>
-        </div>
+        <span
+          aria-hidden="true"
```

Comment:
hm... why is the link displayname aria-hidden at all? 🤔

Reply:
Well that's a good question 😅 I can't think of any good reason to be honest... should I remove it?

Reply:
Yes, please 👍